src/Aesthetic.ts

```diff
@@ -688,15 +689,16 @@ export const dimensions = {
   y : Y
 };
 
-type concrete_aesthetics = X | 
-Y |
-Size | 
-Jitter_speed |
-Jitter_radius | 
-Color | 
-Filter;
+type ConcreteAesthetic = 
+  | X
+  | Y
+  | Size
+  | Jitter_speed
```

Comment:
What's up with this formatting, especially the line before the first X? I'm conditioned to think of leading lines as guard rather than an OR.

Reply:
It's the same formatting that OCaml uses - I prefer it for multi-line union types as it keeps everything consistent.